uwblueprint/website-bp-be
# Created a sample GraphQL request

## Notion ticket link

[GraphQL Request: Make a GraphQL request in the BE repository that can be called via Postman. ](https://www.notion.so/uwblueprintexecs/GraphQL-Request-Make-a-GraphQL-request-in-the-BE-repository-that-can-be-called-via-Postman-52e1ab9f856b4a3a95c0c38839ef343a)



## Implementation description
* Added a test type and resolver to handle finding a member in the **`members.json`** file



## Steps to test
1. Locate the backend directory and run using `yarn dev`
2. Send a request to `http://localhost:5000/graphql` with the following query
```
query {
  getPerson(name: "Your Name") {
    name
    role
    term
    teams
    img
  }
}
```
3. Your information should be returned


## What should reviewers focus on?
* File Structure
    * Sample data directory


## Checklist
